<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFilt</class>
 <widget class="QDialog" name="DialogFilt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Low/High Pass Fir Filter Design</string>
  </property>
  <widget class="QTextEdit" name="TextSampleRate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sample Rate Samples/Sec  Fs</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextCutoff">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Cutoff Frequency Hz</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextTransitionFreq">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Transition  Freq Hz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalcButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Calc FIR</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditFileName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditStopBand">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stop Band dBv</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditPassRipple">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pass Band Ripple %</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="FirWindowHamming">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hamming</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="FirwindowBlackman">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Blackman</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="FirWindowHann">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hann</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FIR Window</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBLP">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Pass</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioBHP">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>High Pass</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
